app/views/volunteers/edit.html.slim

```diff
@@ -2,7 +2,12 @@
   .col-xs-12
     h1= t_title(:edit)
 
-== render 'form'
+.row
+  .col-xs-12
+    = simple_form_for(@volunteer) do |f|
```

Comment:
Im stating this, because I realized that all of us (including myself) where extensiveley nesting things. At first this works just fine, but as soon these nestings are getting messed up (they will), it tends to get hairy...